coffeecard/CoffeeCard.Library/Services/PurchaseService.cs

```diff
@@ -63,6 +64,20 @@ public Purchase RedeemVoucher(string voucherCode, IEnumerable<Claim> claims)
             return purchase;
         }
 
+        private async Task<Guid> GenerateUniqueOrderId()
```

Comment:
Normally I would say refactor to avoid code duplication here, but do we want to care considering it will likely be deprecated soon? I'm guessing that's why you didn't?

Reply:
The old purchase service will be deprecated as soon as we app is not using that Endpoint